.vscode/cspell.json

```diff
@@ -37,49 +46,373 @@
     "*.certificate",
     "*.crt",
     "*.yml",
-    ".vscode/cspell.json"
+    ".vscode/cspell.json",
+    "*.parquet",
+    "**/sdk/**/src/main/java/**/implementation/**",
+    "**/*-perf/**",
+    "**/*-tests/**",
+    "**/*-test-*/**",
+    "**/resourcemanager/**",
+    "**/azure-resourcemanager*/**",
+    "**/microsoft-azure-*/**",
+    "**/azure-core-jackson-tests/**",
+    "**/src/test/**",
+    "**src/**/samples/**/*.json",
+    "**/src/**/samples/**/*.ipynb",
+    "**/sdk/**/swagger/**",
+    "**/src/**/resources/**",
+    "sdk/aot/azure-aot-graalvm-samples/**",
+    "sdk/agrifood/azure-verticals-agrifood-farming/**",
+    "sdk/aot/azure-aot-graalvm-perf/**",
```

Comment:
I missed this one in the last review. Since we're already excluding `**/*-perf/**` we can remove entries in here that have `-perf/` in the name.